app/common/runtimeTypes/storage/wallets.ts

```diff
@@ -38,6 +38,7 @@ export const NewMnemonicsSuccess = t.intersection([
 export type NewMnemonicsSuccess = t.TypeOf<typeof NewMnemonicsSuccess>
 
 export const newMnemonicsErrors = {
+  GENERIC_ERROR: t.literal("GENERIC_ERROR"),
```

Comment:
I think that the `GENERIC_IPC_ERROR` exported in app/common/runtimeTypes/ipc/index.ts could be used at this point. There is an example in getWalletErrors (app/common/ipc/wallets.ts). I also think that all related IPC types and consts (NewMnemonicsSuccess, newMnemonicsErrors, NewMnemonicsErrors) should be placed in app/common/runtimeTypes/ipc/wallets.ts, not in storage rt-types.

Reply:
I think that this should be done in another PR solving their own issue